packages/suite/src/views/wallet/transactions/TradeBox/TradeBoxMenu.tsx

```diff
@@ -0,0 +1,133 @@
+import { Route } from '@suite-common/suite-types';
+import { Account } from '@suite-common/wallet-types';
+import { Button, Dropdown } from '@trezor/components';
+import { FONT_SIZE, FONT_WEIGHT } from '@trezor/components/src/config/variables';
+import { EventType, analytics } from '@trezor/suite-analytics';
+import { ReactNode } from 'react';
+import { goto } from 'src/actions/suite/routerActions';
+import { Translation } from 'src/components/suite';
+import { OverflowItems } from 'src/components/suite/OverflowItems';
+import { useDispatch } from 'src/hooks/suite';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+    & > * {
+        margin-right: 12px;
+    }
+`;
+
+const StyledDropdown = styled(Dropdown)<{ isDisabled: boolean }>`
+    background: ${({ isDisabled, theme }) => (isDisabled ? theme.BG_GREY : theme.BG_SECONDARY)};
+    width: 38px;
+    height: 38px;
+    border-radius: 8px;
+    margin-right: 0px;
+    flex-shrink: 0;
+
+    > :first-child {
+        width: 100%;
+        height: 100%;
+
+        :hover {
+            background: ${({ isDisabled, theme }) => !isDisabled && theme.BG_SECONDARY_HOVER};
+        }
+    }
+`;
+
+const StyledButton = styled(Button)`
+    font-size: ${FONT_SIZE.NORMAL};
+    font-weight: ${FONT_WEIGHT.DEMI_BOLD};
+    padding-left: 20px;
+    padding-right: 20px;
```

Comment:
Why use custom paddings and fonts here?

Reply:
I see, it's copied from `AppNavigation`. I'd only leave the `font-weight` here and even that is controversial 😅